client/src/app/containers/Game/types.ts

```diff
@@ -0,0 +1,17 @@
+import { User } from '../../../auth/slice';
+
+export interface ActionInputProps {
+    isBlocking: boolean;
+}
```

Comment:
:mag: Where is this used?

Reply:
In every `*Action` component. Maybe this isn't the best place to define it, but it's the base input props type for each of those.